src/styles/empty-view.module.scss

```diff
@@ -0,0 +1,14 @@
+.wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 100%; 
+    overflow: auto; 
+    position: fixed;
+    z-index: 2;
```

Comment:
Can u verify the other z-indexes used in the app. May be u can set a higher value